nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -232,73 +232,73 @@ providers:
           - Private Network (10.0.0.0/8 only)
         labels: *provider_limestone_pools_s1_small_labels
 
-  - name: ovh-bhs3
-    cloud: ovh
-    region-name: BHS3
-    boot-timeout: 300
-    launch-timeout: 300
-    rate: 0.001
-    diskimages: &provider_ovh_diskimages
-      - name: centos-7
-        config-drive: true
-      - name: centos-8
-        config-drive: true
-      - name: fedora-30
-        config-drive: true
-      - name: fedora-31
-        config-drive: true
-      - name: ubuntu-bionic
-        config-drive: true
-      - name: ubuntu-xenial
-        config-drive: true
-    pools:
-      # NOTE(pabelanger): Please contact pabelanger before making changes to
-      # this pool, especially increasing max-servers or changing labels. There
-      # is a real world financial cost in doing so that is charged to the
-      # Ansible org.
-      - name: main
-        max-servers: 0
-        labels: &provider_ovh_pools_main_labels
-          - name: centos-7-1vcpu
-            flavor-name: s1-4
-            diskimage: centos-7
-            key-name: infra-root-keys
-          - name: centos-8-1vcpu
-            flavor-name: s1-4
-            diskimage: centos-8
-            key-name: infra-root-keys
-          - name: fedora-30-1vcpu
-            flavor-name: s1-4
-            diskimage: fedora-30
-            key-name: infra-root-keys
-          - name: fedora-31-1vcpu
-            flavor-name: s1-4
-            diskimage: fedora-31
-            key-name: infra-root-keys
-          - name: ubuntu-bionic-1vcpu
-            flavor-name: s1-4
-            diskimage: ubuntu-bionic
-            key-name: infra-root-keys
-          - name: ubuntu-xenial-1vcpu
-            flavor-name: s1-4
-            diskimage: ubuntu-xenial
-            key-name: infra-root-keys
-
-  - name: ovh-uk1
-    cloud: ovh
-    region-name: UK1
-    boot-timeout: 300
-    launch-timeout: 300
-    rate: 0.001
-    diskimages: *provider_ovh_diskimages
-    pools:
-      # NOTE(pabelanger): Please contact pabelanger before making changes to
-      # this pool, especially increasing max-servers or changing labels. There
-      # is a real world financial cost in doing so that is charged to the
-      # Ansible org.
-      - name: main
-        max-servers: 0
-        labels: *provider_ovh_pools_main_labels
+#  - name: ovh-bhs3
```

Comment:
good idea, we never enable this cloud. Lets just delete it for now, until we can properly onboard.